token-service/src/main/java/org/p2p/token/service/model/TokenServiceMetadata.kt

```diff
@@ -0,0 +1,9 @@
+package org.p2p.token.service.model
+
+data class TokenServiceMetadata(
+    val address: String,
```

Comment:
String -> Base58String

same question about the `address`. We use mintAddress all the time, why change now?